.env

```diff
@@ -0,0 +1,4 @@
+# mongo "mongodb+srv://cluster0-wojc5.mongodb.net/test"  --username user-one
+# mongodb://username:password@host:port/database?options...
+MONGODB_URI = mongodb+srv://user-one:password%21@cluster0-wojc5.mongodb.net/test?retryWrites=true&w=majority
```

Comment:
@william000000, good job so far, but I think that these should not be here. In fact, the whole file should not be here. consider placing it in the `.gitignore` file and we're good to go!